lib/environment.js

```diff
@@ -56,10 +57,21 @@ class NunjucksEnvironment extends nunjucks.Environment {
       const filterPath = path.join(unit.path, 'app/extend/filter.js');
       const filters = this.app.loader.loadFile(filterPath) || {};
       for (const name of Object.keys(filters)) {
-        this.addFilter(name, filters[name]);
+        this.addFilter(name, this[FILTER_WRAPPER](filters[name]), true);
```

Comment:
I think, we could check fn style here, then decide whether wrap it.

use https://github.com/node-modules/is-type-of